readthedocs/organizations/models.py

```diff
@@ -69,6 +69,11 @@ class Organization(models.Model):
         help_text='Docs and builds are disabled for this organization',
         default=False,
     )
+    max_concurrent_builds = models.IntegerField(
```

Comment:
I think we want this to come from the organizations plan, not from the organization itself. We can have overrides on the org, but in general it should come from the Plan. We need to think about how to architect this, so it's easy to override on the .com.

Reply:
Yes, I agree. The max per organization should come from the organization's plan. We could:

1. update the `organization.max_concurrent_builds` via a signal when the organization's plan changes (listening to `Organization.post_save` or similar) to keep this value updated
1. write `QuerySet`/`Manager` (similar to [`CorporateProjectQuerySet.is_active`](https://github.com/readthedocs/readthedocs-corporate/blob/8a8dc755631e5e4513f830836378fa02865fdbc5/readthedocsinc/projects/querysets.py#L80-L101)) method that's overridden in .com to return `org.max_concurrent_builds or plan.max_concurrent_builds` 

1) sounds more like a hack and may be hard to keep them in sync if something fails (or if we change the max per org at a Plan level. 2) looks more similar to the pattern we have been following, so it's probably better.

Reply:
Yea, I'd prefer #2. No reason to create a cache to have issues with if you can query the value directly.

Reply:
Great! I updated this code to use a QuerySet and created a PR in commercial to override this and consider the plan limits. Let me know!